src/SprykerEco/Service/ComputopApi/Mapper/ComputopApiMapper.php

```diff
@@ -150,7 +150,7 @@ public function generateTransId(QuoteTransfer $quoteTransfer): string
     {
         $parameters = [
             $this->createUniqueSalt(),
-            $quoteTransfer->getCustomer()->getCustomerReference(),
+            uniqid(),
```

Comment:
Is uniqueid() needed? According to PHP other functions could include a bit more entropy these days.
See https://www.php.net/manual/de/function.uniqid.php

Reply:
I will add $moreEntropy = true. Other functions are good for cryptography  but we do not need such functionality. 
Previously `$quoteTransfer->getCustomer()->getCustomerReference()` always returned null, so there was no uniqueness.